elements/lisk-transactions/src/utils/validation/schema.ts

```diff
@@ -65,6 +65,64 @@ export const transaction = {
 	},
 };
 
+export const transactionInterface = {
+	required: [
+		'toJSON',
+		'isReady',
+		'getBytes',
+		'validate',
+		'verifyAgainstOtherTransactions',
+		'apply',
+		'undo',
+		'prepare',
+		'addMultisignature',
+		'addVerifiedMultisignature',
+		'isExpired',
+		'fromSync',
+	],
+	properties: {
+		toJSON: {
+			typeof: 'function',
```

Comment:
Please add one test case to check this particular keyword `typeof`, or another new keyword specifically used in this schema.

Reply:
Addressed. Good catch =)